tendermint-rs/tests/support/rpc/abci_query.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Unlike the other JSON examples found in this directory, this one wasn't taken live from a `gaiad` RPC response, but is copy/paste from https://tendermint.com/rpc/#abciquery